video/out/vulkan/context_display.c

```diff
@@ -424,6 +424,14 @@ static bool display_init(struct ra_ctx *ctx)
         .alphaMode = VK_DISPLAY_PLANE_ALPHA_OPAQUE_BIT_KHR,
     };
 
+    PFN_vkCreateDisplayPlaneSurfaceKHR vkCreateDisplayPlaneSurfaceKHR =
+        (PFN_vkCreateDisplayPlaneSurfaceKHR)vkGetInstanceProcAddr(vk->vkinst->instance,
+                                                                  "vkCreateDisplayPlaneSurfaceKHR");
+    if (!vkCreateDisplayPlaneSurfaceKHR) {
+        MP_ERR(ctx, "VK_KHR_display is not supported by the driver!\n");
+        goto error;
+    }
+
```

Comment:
This also needs to be done for a few calls in `walk_display_properties`: `vkGetPhysicalDeviceDisplayPropertiesKHR`, `vkGetPhysicalDeviceDisplayPlanePropertiesKHR`, `vkGetDisplayPlaneSupportedDisplaysKHR`, and `vkGetDisplayModePropertiesKHR`.

Also, I think it'd be preferable to give the local variables names that don't shadow the global symbols.

Reply:
It's worth pointing out that while we need to load all functions, we only need one *check*

Reply:
Also this may help

```c
#define VK_LOAD_FUN(vk, name) \
    PFN_vk##name name = (PFN_vk##name) vk->vkinst->get_proc_addr(vk->vkinst->instance, "vk" #name);
```

Usage `VK_LOAD_FUN(vk->vkinst->instance, CreateDisplayPlaneSurfaceKHR